.circleci/soltest.sh

```diff
@@ -0,0 +1,96 @@
+#! /bin/bash
```

Comment:
I'm a bit uneasy about duplicating some of `scripts/tests.sh` here - especially since nothing from `scripts/tests.sh` is removed. For example, we do not need the error reporting feature there anymore.

Reply:
Also functions like `run_aleth` could be moved into a common utils file.